scout/server/links.py

```diff
@@ -63,6 +63,24 @@ def add_gene_links(gene_obj, build=37):
     gene_obj["cbioportal_link"] = cbioportal_gene(hgnc_symbol)
     gene_obj["civic_link"] = civic_gene(hgnc_symbol)
     gene_obj["iarctp53_link"] = iarctp53(hgnc_symbol)
+    gene_obj["stripy_link"] = stripy_gene(hgnc_symbol)
+    gene_obj["gnomad_str_link"] = gnomad_str_gene(hgnc_symbol)
+
+
+def stripy_gene(hgnc_symbol):
+    link = "https://stripy.org/database/{}"
+
+    if not hgnc_symbol:
+        return None
+    return link.format(hgnc_symbol)
+
+
+def gnomad_str_gene(hgnc_symbol):
+    link = "https://gnomad.broadinstitute.org/short-tandem-repeat/{}?dataset=gnomad_r3"
```

Comment:
This one also: 👍🏻 

<img width="200" alt="image" src="https://user-images.githubusercontent.com/28093618/155958983-3ffe01a2-4a71-42ba-978e-52dcb315fe17.png">